presto-main/src/main/java/com/facebook/presto/sql/planner/plan/ExchangeNode.java

```diff
@@ -57,7 +55,18 @@
     public enum Scope
```

Comment:
I don't think scope makes so much sense as a name anymore since it also indicates whether it's materialized, but I don't have a better suggestion, since `Type` is already taken.

Reply:
`Channel` as I suggested in https://github.com/prestodb/presto/pull/12419#pullrequestreview-209896190 ;) 

Although I feel this would make a lot of changes. So maybe we can just keep the legacy `Scope` name here ....

Reply:
If we also rename the enum values as  you suggested, channel seems fine to me.  But as you said that would be a lot of changes (though not hard to do since intellij can do it for you).  for now I'm ok leaving "Scope" until the usage settles a bit.